webextensions/api/identity.json

```diff
@@ -12,9 +12,15 @@
               "edge": {
                 "version_added": "79"
               },
-              "firefox": {
-                "version_added": "53"
-              },
+              "firefox": [
+                {
+                  "version_added": "53"
+                },
+                {
+                  "version_added": "86",
+                  "notes": "The loopback address http://127.0.0.1/mozoauth2/[subdomain of URL returned by identity.getRedirectURL()] can be used as the redirect URL to enable the use of Google OAuth flows."
```

Comment:
This level of detail is too much. Let's refer to the inline documentation instead, like this:

```html
The <code>redirect_url</code> parameter now supports a loopback address, see <a href="https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/identity#getting_the_redirect_url">Getting the redirect URL</a>.
```